### DISCUSSION TITLE
Wide View Mode: horizontally

### DISCUSSION THREAD

**Discussion by MilkyFake** [YU000jp/logseq-page-tags-and-hierarchy#43] [Q&A]
everything from the attached screenshot does not work on version 2.17, the location does not change:(
![image](https://github.com/YU000jp/logseq-page-tags-and-hierarchy/assets/98526507/fc295f66-768f-4d60-9247-458cce8de704)

**Comment by MilkyFake** [comment]
upd:
 I do not know what this is related to, but these 3 guys have learned to move among themselves, (by the way, there was only 1 guy there before (unlinked references)) so, when you change the settings of these guys, they are amenable to change only up to 3, from that they move only along the x axis and nowhere else, so they move among themselves in the same plane:\
![image](https://github.com/YU000jp/logseq-page-tags-and-hierarchy/assets/98526507/7e136c9d-14c0-4989-b891-5e15d4f06b6d)
![image](https://github.com/YU000jp/logseq-page-tags-and-hierarchy/assets/98526507/1444db22-b0b1-42bf-b8c1-7fe0bcb2bf8b)
![image](https://github.com/YU000jp/logseq-page-tags-and-hierarchy/assets/98526507/07655fd2-b31f-4509-9bee-2063a0b70c24)
and also if you turn them all over ( by clicking on them ) the page will start to stretch fiercely ( the screenshot shows how the page scroll slider appeared when I turned these guys over )
![image](https://github.com/YU000jp/logseq-page-tags-and-hierarchy/assets/98526507/65b8aa71-66df-4c4c-919c-777cf73931aa)

**Comment by YU000jp** [comment]
That explanation was missing. They only move horizontally. The leftmost number is 1 and increases by 1 to the right. Depending on the page, they may or may not be visible.
